README.md

```diff
@@ -365,6 +365,49 @@ private Binding<HttpStatus> problems(final Condition<HttpStatus> condition) {
 }
 ```
 
+#### Template Routing
+
+Routing and response handling is sometimes pretty similar for many request, and can even be handle using the same
+defined routing logic for different requests. To enable reuse of this routing logic, Riptide supports the creation
+of nested *template routing*, that can be reused for dispatching different request. The following example defines
+a nested hierarchy of routing templates:
+
+```java
+static final Router<MediaType> CONTENT_ROUTER = Router.create(contentType(),
```

Comment:
The whole pull-request works on the assumption that it's expensive to re-create a routing tree for every invocation. Is that the case? Can we measure both versions? It pretty much sounds like premature optimization to me.